components/Icons.es6.js

```diff
@@ -473,9 +473,9 @@ function Close ({ color, className, path, viewBox, width, height, style }) {
   )
 }
 
-function ImageUpload ({ color, className, path, viewBox, width, height, style }) {
+function ImageUpload ({ color, className, path, width, height, style }) {
```

Comment:
The only place that `ImageUpload` is being used is in `common/js/media/Empty.es6.js`. I feel that we should fix all `viewBox` parameters to its original, because there isn't a need to modify the viewbox in most, if not all cases, and it makes using the icon component confusing to use in the first place (you would need to know what the viewBox value should be so that the icon can properly fit into the svg container). By setting the viewbox to a fixed value, anyone wanting to use the `ImageUpload` icon can just use it, and modifying its size through the width and height.

Reply:
Here is another place where we use `https://github.com/b12io/metronome/blob/1c8a1b11f3d3391fbb5a3de2d2d97dc2fe91754e/components/form/file-upload/FileUpload.es6.js#L82`

I believe we use FileUpload in multiple places in product. Changing the viewbox will change the look and feel of it in many places.

Reply:
Makes sense. I'll revert it then!